lib/starknet_explorer_web/live/pages/home/index.ex

```diff
@@ -171,10 +171,12 @@ defmodule StarknetExplorerWeb.HomeLive.Index do
 
   @impl true
   def handle_info(:load_blocks, socket) do
+    [latest_block | blocks] = Block.latest_n_blocks_with_txs(15)
```

Comment:
Here we should do something like

```
blocks = Block.latest_n_blocks_with_txs(15)
latest_block = List.first(blocks)
```
Otherwise the Html will render 14 blocks instead of 15 because the first one is not included in the `blocks` list anymore.

Reply:
[Fixed](https://github.com/lambdaclass/madara_explorer/pull/28/files#diff-c52564ce8d34b79e1f03a77c7324e3139a44a1fb0911664eeb03c3f2ced03f0eR227)